tv_apps/browser/js/sync/settings.js

```diff
@@ -261,6 +295,13 @@
         navigator.mozId.watch({
           wantIssuer: 'firefox-accounts',
           onlogin: () => {
+            // This callback is fired everytime a new account logs in,
+            // but we only want to enable the history setting when this
+            // callback is called because the refresh auth flow succeeded.
```

Comment:
This code seems to be present in `onhistorychecked`, but not in `onbookmarkschecked` - why this asymmetry?